packages/mdc-list/README.md

```diff
@@ -77,6 +77,8 @@ in the double line list style as defined by
 </ul>
 ```
 
+> NOTE: Make sure there are no white-space characters before primary and secondary text content.
```

Comment:
Is there another way to implement these mixins that doesn't choke on whitespace?

I suspect a lot of people are going to run into this, and it's going to take them a while to figure out why their UI suddenly broke.

Reply:
I would like there to be a way, but flex-box was the one way we figured...but that causes the current problem where IE and Edge don't work correctly.

The odd bit is if anything, I'd say IE and Edge are working as I'd expect all browsers to. It's more of a mystery to me why switching to flex still worked in the others.

Reply:
Since the baseline struts (before & after pseudo elements) use `display: inline-block` it leaves [extra space](https://css-tricks.com/fighting-the-space-between-inline-block-elements/) around before the text when white-space characters are inserted. We avoided this problem by using flex box before but IE / Edge renders them differently on pseudo elements.

It shouldn't create any issues for users who are using frameworks while vanilla component users should use this with caution.